services/order.py

```diff
@@ -0,0 +1,37 @@
+from django.db import transaction
+from django.db.models import QuerySet
+from django.contrib.auth import get_user_model
+from db.models import Order, Ticket, MovieSession
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> None:
+    with transaction.atomic():

```

Comment:
In case you make a whole function body as an atomic transaction, you could use `transaction.atomic` as a decorator